src/locales/en/common.json

```diff
@@ -58,6 +58,7 @@
     "hideBtn": "Hide answers",
     "text": "Share your thoughts with others",
     "answers": "Count of answers",
-    "isAdmin": "Manager"
+    "isAdmin": "Manager",
+    "error": "Field should contain from 2 to 700 characters"
```

Comment:
please, check src/locales/en/error.json for this field, same for ua

Reply:
Before adding any translation, try to find that translation with a global search. And if only there are no results you can add new one :).

Reply:
fixed :)